sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -382,19 +402,15 @@ aforementioned command is a Job ID.
 
 The only way to see the results of such a job is with the `sem logs` command.
 
-## The sem stop command
-
-`sem` gives you the capability to stop a running `job` or a running `pipeline`
-without the need to visit the Semaphore 2.0 UI with the help of the `sem stop`
-command.
+### sem stop
 
-The `sem stop` command works as follows:
+You can stop a running job or pipeline with `sem stop`:
 
     sem stop job|pipeline [ID]
 
-If you are executing `sem stop job`, you will need to provide the `ID` of a
-running job. Similarly, if you are executing `sem stop pipeline`, you will need
-to provided the `ID` of a running pipeline.
+If you are executing `sem stop job`, you will need to provide an `ID` of a
+running job. If you are executing `sem stop pipeline`, you will need
+to provide an `ID` of a running pipeline.
```

Comment:
I think that `**the** ID of a running pipeline` is better.

Reply:
Not sure, I think we need a copyeditor for details like this anyway. :)